_pytest/python.py

```diff
@@ -1163,6 +1165,56 @@ def __exit__(self, *tp):
         self.excinfo.__init__(tp)
         return issubclass(self.excinfo.type, self.ExpectedException)
 
+# builtin pytest.warns helper
+
+def warns(ExpectedWarning, *args, **kwargs):
```

Comment:
I was just following the style of `raises`, above. This function definitely needs a docstring, though.